### PR TITLE
Update .dockerignore

### PR DESCRIPTION
**Pull request by superpoussin22**
remove duplicat lines

## Title

Remove duplicate lines in .dockerignore

## Type

🧹 Refactoring

## Changes

simple QA

### CODE REVIEW

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
what is this pr for @superpoussin22 ?

**Comment by superpoussin22** [comment]
@ishaan-jaff  the .dockerignore  previously contains duplicated lines . i remove the duplicates

**Comment by ishaan-jaff** [comment]
@superpoussin22 I don't see any changes to the docker ignore in this PR, maybe the wrong files got edited here

File: litellm/caching.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def set_cache(self, key, value, **kwargs):
     async def async_set_cache(self, key, value, **kwargs):
         self.set_cache(key=key, value=value, **kwargs)
 
-    async def async_set_cache_pipeline(self, cache_list, ttl=None):
+    async def async_set_cache_pipeline(self, cache_list, ttl=None, **kwargs):
```
Comment (ghost) [inline litellm/caching.py:126]: why is this PR changing caching.py?  @superpoussin22

Comment (superpoussin22) [reply litellm/caching.py:126]: @krrishdholakia  the commit for caching is not done by me if you look at the commits in the PR that's strange
and I see the changes in .docker ignore so better closing the PR

**Pull request closed without merging**